### ISSUE TITLE
Tudo novo?

### ISSUE THREAD

**Issue by renatobenks** [OndaDura/church-app#1]
Hey mano, vamos começar tudo novo no back-end agora?

**Comment by matheuspoleza** [comment]
sim! Vou programar a parte que está sendo requirida de patrimonio primeiramente, vou passar para node.js, javascript no server. Vai ficar bem fácil, essa primeira parte eu vou puxar e depois passo treinamento

**Comment by renatobenks** [comment]
Opaaaaa! Beleza mano. Vai ficar mais fácil pra mim poder ajudar já que tenho conhecimentos a cerca disso já. Vamos usar Mongo?

**Comment by matheuspoleza** [comment]
@renatobenks provavelmente, é muito fácil mexer node.js com mongo

**Comment by renatobenks** [comment]
Concordo plenamente! Haha
